lib/addons/eks-managed-core-addons/index.ts

```diff
@@ -0,0 +1,30 @@
+import { CfnAddon } from "@aws-cdk/aws-eks";
+import { ClusterAddOn } from "../..";
+import { ClusterInfo } from "../../spi";
+
+export class EksManagedCoreAddOnProps {
+
+    readonly addOnName: string;
+    readonly version: string;
+}
+
+/**
+ * Implementation of EKS Managed add-ons.
+ */
+export class EksManagedCoreAddOns implements ClusterAddOn {
```

Comment:
We discussed either managed or core. I see you went with both. It is fairly minor, but from what I remember the EKS service team decided to drop the work "managed". So let's go with "core".